recipes/qcodes/meta.yaml

```diff
@@ -0,0 +1,64 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "qcodes" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "a70f3a209e2f4371634b12094def37696fa2accb2f0746e5dffba50e4918e33e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/QCoDeS/Qcodes/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
```

Comment:
The `README` says this package is compatible with python 3.5+, but the `setup.py` pypi metadata indicates 3.3, 3.4, and 3.5 are supported. The docs say only 3.4 and 3.5.